<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnggDiffractionQtTabSettings</class>
 <widget class="QWidget" name="EnggDiffractionQtTabSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Calibration settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_calib_settings">
        <property name="title">
         <string>Input directories/folders:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Calibration:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_input_dir_calib">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_browse_input_dir_calib">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Raw files:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_input_dir_raw">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_browse_input_dir_raw">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>label</zorder>
        <zorder>lineEdit_input_dir_calib</zorder>
        <zorder>pushButton_browse_input_dir_raw</zorder>
        <zorder>lineEdit_input_dir_raw</zorder>
        <zorder>pushButton_browse_input_dir_calib</zorder>
        <zorder>label_2</zorder>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_calib_adv_settings">
        <property name="title">
         <string>Advanced settings:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_template_gsas_prm">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushButton_browse_template_gsas_prm">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Template .prm file:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QGroupBox" name="groupBox_rebing">
           <property name="title">
            <string>Rebin for Calibrate (and Focus)</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <property name="margin">
             <number>3</number>
            </property>
            <property name="spacing">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox_rebin_size_calib">
                <property name="maximumSize">
                 <size>
                  <width>120</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>-999999999.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.000001000000000</double>
                </property>
                <property name="value">
                 <double>-0.000500000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Time-of-Flight</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="checkBox_force_recalculate_overwrite">
             <property name="toolTip">
              <string>Recalculate the vanadium curve and per-pixel integration even if files are available with results previously calculated</string>
             </property>
             <property name="text">
              <string>Force recalculate all existing Vanadium files</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_pixel_calib">
        <property name="title">
         <string>Pixel calibration</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>File name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_pixel_calib_filename">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_browse_pixel_calib_filename">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>275</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
